src/cli/bruker2dicom

```diff
@@ -58,13 +61,34 @@ def main():
     logging.getLogger().setLevel(verbosity.upper())
     
     function = arguments.pop("function")
+    source = arguments.pop("source")
+    
+    #create tmp directory in the current location
+    extractdir = tempfile.mkdtemp("","",".")
```

Comment:
Creating a temporary directory in the current working directory (`"."`) is usually a bad idea: we have no way to ensure that this is writable. Also, it is not necessary to remove the `prefix` argument. The line should read:
```python
extractdir = tempfile.mkdtemp()
```